react-function-components/src/index.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+function CustomButton() {
+  return <button>Click Me!</button>;
+}
+
+const element = <CustomButton />;
+
+ReactDOM.render(
+  element,
```

Comment:
No real need to save it into a `const` first, you can just pass the JSX element directly to `render`